### PR TITLE
Add VSCodium and Verdaccio

### PR DESCRIPTION
**Pull request by horosgrisa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -546,13 +546,15 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [uTorrent](http://www.utorrent.com/)
 - [ulauncher](https://ulauncher.io/)
 - [Ventrilo](http://www.ventrilo.com/)
+- [Verdaccio](https://github.com/verdaccio/verdaccio)
```
Comment (lra) [inline README.md]: Can you use https://verdaccio.org/ instead?

File: README.md [hunk 2]
```diff
@@ -546,13 +546,15 @@ See the [README](doc/README.md) file in the doc directory for more info.
...
 - [Versions](http://www.versionsapp.com)
 - [Vim](http://www.vim.org/)
 - [Vimperator](http://www.vimperator.org/vimperator)
 - [Viscosity](http://www.sparklabs.com/viscosity/)
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Visual Studio Code - Insiders](https://code.visualstudio.com/insiders)
 - [Visual Studio Code - OSS](https://github.com/Microsoft/vscode)
+- [VSCodium](https://github.com/VSCodium/vscodium)
```
Comment (lra) [inline README.md]: Can you use https://vscodium.com/ ?

File: mackup/applications/verdaccio.cfg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[application]
+name = Verdaccio
+
+[configuration_files]
+
+.config/verdaccio/config.yaml
+
```
Comment (lra) [inline mackup/applications/verdaccio.cfg]: unneeded newline

File: mackup/applications/verdaccio.cfg [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[application]
+name = Verdaccio
+
+[configuration_files]
+
```
Comment (lra) [inline mackup/applications/verdaccio.cfg]: unneeded newline

**Comment by horosgrisa** [comment]
@lra Check now

**Pull request merged**